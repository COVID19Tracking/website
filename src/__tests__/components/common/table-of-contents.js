import React from 'react'
import renderer from 'react-test-renderer'
import TableOfContents from '~components/common/table-of-contents'

describe('Components : Common: Table of Contents', () => {
  it('renders correctly', () => {
    const tree = renderer
      .create(
        <TableOfContents
          headings={[
            { id: 'a', value: 'A', depth: 1 },
            { id: 'b', value: 'B', depth: 2 },
          ]}
        />,
      )
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})
