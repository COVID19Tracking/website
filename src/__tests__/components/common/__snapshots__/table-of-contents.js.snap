// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Components : Common: Table of Contents renders correctly 1`] = `
<div
  className="tableOfContents"
>
  <h2>
    Table of contents
  </h2>
  <ul>
    <li
      className={false}
    >
      <strong>
        A
      </strong>
    </li>
    <li
      className="subItem"
    >
      <a
        href="#b"
      >
        B
      </a>
    </li>
  </ul>
</div>
`;
